FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -221,6 +248,29 @@ - (void)appDidBecomeActiveNotification:(NSNotification *)notification {
       } else {
         [self.appStartTrace cancel];
       }
+
+      if ([self prewarmAvailable] &&
+          (currentTimeSinceEpoch - startTimeSinceEpoch < gAppStartMaxValidDuration)) {
+        BOOL activePrewarm = [self isActivePrewarm];
+        BOOL doubleDispatchPrewarm = [self isDoubleDispatchPrewarm];
+        if (!activePrewarm && !doubleDispatchPrewarm) {
+          [self.coldStartTrace stop];
+          [self.prewarmedStartTrace cancel];
```

Comment:
Why do you need `return`? 
Nit: Can you connect `if (!activePrewarm && !doubleDispatchPrewarm)` and `if (activePrewarm && doubleDispatchPrewarm)` with `else if` for better readability?